src/protocols/bitcoin/components/TransferSendForm.vue

```diff
@@ -0,0 +1,234 @@
+import type {
+  TransferFormModel,
+} from '@/types';
+import { BTC_SYMBOL } from '@/protocols/bitcoin/config';
+import { useTransferSendForm } from '@/composables/transferSendForm';
+import { PROTOCOL_BITCOIN } from '@/constants';
+
+import { INFO_BOX_TYPES } from '@/popup/components/InfoBox.vue';
+import DetailsItem from '@/popup/components/DetailsItem.vue';
+import TransferSendFormBase from '@/popup/components/TransferSendFormBase.vue';
+import TransferSendRecipient from '@/popup/components/TransferSend/TransferSendRecipient.vue';
+import TransferSendAmount from '@/popup/components/TransferSend/TransferSendAmount.vue';
+import TransactionSpeedPicker from '@/popup/components/TransactionSpeedPicker.vue';
+
+import EditIcon from '@/icons/pencil.svg?vue-component';
+import DeleteIcon from '@/icons/trash.svg?vue-component';
+import PlusCircleIcon from '@/icons/plus-circle-fill.svg?vue-component';
+
+export default defineComponent({
+  name: 'BtcTransferSendForm',
+  components: {
+    TransactionSpeedPicker,
+    DetailsItem,
+    TransferSendAmount,
+    TransferSendRecipient,
+    TransferSendFormBase,
+  },
+  model: {
+    prop: 'transferData',
+  },
+  props: {
+    transferData: { type: Object as PropType<TransferFormModel>, required: true },
+  },
+  setup(props, { emit }) {
+    const store = useStore();
+
+    const hasMultisigTokenWarning = ref(false);
+    const isUrlTippingEnabled = ref(false);
+
+    const { balance } = useBalances({ store });
+    const {
+      activeAccount,
+    } = useAccounts({ store });
+
+    const {
+      formModel,
+      errors,
+      hasError,
+      invoiceId,
+      invoiceContract,
+      clearPayload,
+      openScanQrModal,
+      handleAssetChange,
+    } = useTransferSendForm({
+      transferData: props.transferData,
+    });
+
+    // TODO - fee set proper values
+    const fee = ref(new BigNumber(0));
+    const feeList = [
+      { fee: new BigNumber(0.3), time: 25 },
+      { fee: new BigNumber(0.2), time: 600 },
+      { fee: new BigNumber(0.1), time: 3540 },
+    ];
+
+    const numericFee = computed(() => +fee.value.toFixed());
+
+    function setFee(value: BigNumber) {
+      fee.value = value;
+    }
+
+    function emitCurrentFormModelState() {
+      const inputPayload: TransferFormModel = {
+        ...formModel.value,
+        fee: fee.value as BigNumber,
+        total: numericFee.value + +(formModel.value?.amount || 0),
+        invoiceId: invoiceId.value,
+        invoiceContract: invoiceContract.value,
+      };
+      emit('update:transferData', inputPayload);
+      return nextTick();
+    }
+
+    // Method called from a parent scope - avoid changing its name.
+    async function submit() {
+      if (!hasError.value) {
+        await emitCurrentFormModelState();
+        emit('success');
+      }
+    }
+
+    watch(
+      () => hasError.value,
+      (val) => emit('error', val),
+      { deep: true },
+    );
+
+    watch(
+      () => formModel.value,
```

Comment:
```suggestion
      formModel,
```
No need to use callback with refs.